src/test/java/com/uwetrottmann/tmdb2/TmdbTestSuite.java

```diff
@@ -0,0 +1,218 @@
+package com.uwetrottmann.tmdb2;
+
+import com.googlecode.junittoolbox.SuiteClasses;
+import com.googlecode.junittoolbox.WildcardPatternSuite;
+import com.uwetrottmann.tmdb2.entities.Account;
+import com.uwetrottmann.tmdb2.entities.BaseMovie;
+import com.uwetrottmann.tmdb2.entities.FavoriteMedia;
+import com.uwetrottmann.tmdb2.entities.ListCreateRequest;
+import com.uwetrottmann.tmdb2.entities.ListCreateResponse;
+import com.uwetrottmann.tmdb2.entities.ListOperation;
+import com.uwetrottmann.tmdb2.entities.RatingObject;
+import com.uwetrottmann.tmdb2.entities.WatchlistMedia;
+import com.uwetrottmann.tmdb2.enumerations.AuthenticationType;
+import com.uwetrottmann.tmdb2.enumerations.MediaType;
+import org.assertj.core.util.Strings;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.rules.ExternalResource;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.ArrayList;
+
+import static com.uwetrottmann.tmdb2.TestData.initializeTestingIntegrityData;
+import static com.uwetrottmann.tmdb2.TestData.testList;
+import static com.uwetrottmann.tmdb2.TestData.testListMovies;
+import static com.uwetrottmann.tmdb2.TestData.testMovie;
+import static com.uwetrottmann.tmdb2.TestData.testTvEpisode;
+import static com.uwetrottmann.tmdb2.TestData.testTvSeason;
+import static com.uwetrottmann.tmdb2.TestData.testTvShow;
+
+@RunWith(WildcardPatternSuite.class)
+@SuiteClasses({"**/*Test.class"})
```

Comment:
I Generalized the wildcard to **/*.class 
It checks before if the class is a test class.